### ISSUE TITLE
FAQ: I keep getting "signature verification failed for the transaction" error message on every transaction

### ISSUE THREAD

**Issue by emedrano2112** [pokt-network/pocket-core#1172]
### PURPOSE
The purpose of this issue is meant to clarify why this "signature verification failed for the transaction" error appears, and how to properly solve it when present, without any evident reason.

### GUIDELINES
- If FAQ does not comprehensively cover the issue, community comments and contributions are welcome
- Although the issue is closed, FAQ issues are meant to be maintained and edited over time

## Possible reasons why "signature verification failed for the transaction" error message appears on a transaction
This error is most likely to appear under the following circumstances:
- Pocket has not processed a new block since being started.  IE: `pocket start`
- Pocket has not processed a new block since a validator change:  IE: `pocket accounts set-validator`
in summary.
- The command structure seems to not be valid enough. E.g. `pocket nodes unstake 4e27aeea6248923dec678214ec8fb0508d929e50 0002 100000` 

**Notice**: In this command, the params seem to be OK, but the chainID is "0002" instead of the correct form: `testnet`. In addition, the fee in the command structure is too little. The fee needs to be `10000` not `100000`.

### NOTES:
When attempting to execute a TX (transaction), always note then the TXHash returned, wait for the proper 1-2 blocks to pass and then you can either query your node or query the transaction to get the proper status of the same.
